openlibrary/templates/authors/infobox.html

```diff
@@ -14,6 +14,11 @@
             <td><span itemprop="$itemprop">$value</span></td>
         </tr>
 
+$def render_social_icon(url, icon_url, title):
+    <a href="$url">
+        <img class="profile-icon" src="$icon_url" title="$title">
```

Comment:
```suggestion
    <a href="$url" title="$_('View on %(site)s', site=title)">
        <img class="profile-icon" src="$icon_url" alt="">
```

I think the title is more common `<a>` since that helps screen readers.

Then on the image, you can set `alt=""` to make the image essentially presentation, since the actual info is the `title` (+ remove the title from the image).